response_operations_ui/templates/social.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layouts/social-base.html" %}
+{% block page_title %}Social | Survey Data Collection{% endblock %}
+
+{% block main %}
+  <div role="main" id="main" class="page__main">
+    {% if results is not none%}
+      <h2>
+        {{ results|length }} result{% if results|length != 1 %}s{% endif %} found for {{ postcode }}
```

Comment:
This is appearing at the bottom of the page for me.

![image](https://user-images.githubusercontent.com/28348457/43955296-5a2666b4-9c97-11e8-97ce-b691995099c8.png)

Reply:
I feel there should be further validation that what has been entered is indeed a post code, rather than a bunch of numbers.

Reply:
@benjefferies had a comment on this here: https://github.com/ONSdigital/response-operations-ui/pull/212#discussion_r209232437